### DISCUSSION TITLE
The future...

### DISCUSSION THREAD

**Discussion by raresvanca** [slowscript/warpinator-android#6] [General]
What's the future of this?
I'm curious about how this will be distributed after the release of version 1.0. Or if this could become the official port for Android. (This is hard because the Linux Mint community is not doing android apps), but I just want to know.

What still needs to be implemented or fixed?

**Comment by slowscript** [comment]
I made this for my own use and for the other random people on the internet who like the idea of Warpinator and want an at least somewhat usable android port before the Mint team get to making an official one.

When we get to 1.0, I plan to put it on G Play (already have a developer account) and also possibly F-Droid.
The Mint team have said that they will be making their own ports for other platforms *eventually*. I have no idea whether they will choose to fork this or start from scratch. For now I'm making sure this is marked *unofficial* everywhere.

There is not that much left to implement anymore. That's why I have released the first beta. Out of the bigger things it's just an About dialog and a dialog for choosing a profile picture. Other than that it's just better handling of errors that occur during transfer, finishing localization and some bug fixing.

**Comment by raresvanca** [comment]
It would be cool if they would fork this. But you should tell them you work on this and maybe they would make you the lead developer.
